protobuf/CHANGELOG.md

```diff
@@ -5,7 +5,16 @@
 
   The new limits are consistent with the Java and C++ implementations. ([#1060])
 
+* Fix `GeneratedMessage.addExtension` returning non-frozen and
+  `GeneratedMessage.getExtension` allowing modifying an extension when the
+  message is frozen before initializing the extension field set. ([#1062])
+
+* Fix `GeneratedMessage.getExtension` returning differently typed lists when the
+  message extension field set is initialized and frozen and initialized but not
+  frozen. ([#1062])
```

Comment:
Note: this is not fixing a bug caused by the previous entry, it exists in the master branch independently of the other bugs. However the repro is different in the master branch vs. this branch.